src/authorization/Authorizations.ts

```diff
@@ -389,11 +389,9 @@ export default class Authorizations {
     });
   }
 
-  public static canReadRegistrationToken(loggedUser: UserToken, siteID: string): boolean {
-    return Authorizations.canPerformAction(loggedUser, Entity.TOKEN, Action.READ, {
-      site: siteID,
-      sites: loggedUser.sitesAdmin
```

Comment:
I think that will remove some token handling rights to site admin. Is it intentional?
Have you pushed down that authorization in code paths below in the call stack?

Reply:
@cabch You have to handle the role Site Admin in here + in the Unit Test.
Nevertheless I will merge you PR.
We can have a meeting for this special role.

Reply:
Ok, yes it would be great to clarify that :) thanks
I did it that way as i didn't get the point why I would need sites here